lib/statics/colors.dart

```diff
@@ -18,8 +18,8 @@ class AppColors extends Color {
   /// Tint 90%
   static const primaryWhite = Color(0xffeef1f1);
 
-  /// red[600]
-  static const accentRed = Color(0xffe53935);
+  ///
+  static const accentContrast = Color.fromARGB(255, 247, 149, 3);
```

Comment:
Just an observation- this color seems to fit better than any of the color styles proposed by Figma.